src/main/java/frc/robot/subsystems/intake/IntakeConstants.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.subsystems.intake;
+
+import static edu.wpi.first.units.Units.Radians;
+<<<<<<< HEAD
+=======
+
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+>>>>>>> b38babce7c2115f5fefb513ee61988edc8bcbf20
+
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+
+public class IntakeConstants {
+    public static final double ANGLE_RATIO = 0;
+<<<<<<< HEAD
+    public static MutableMeasure<Angle> DOWN_ANGLE = MutableMeasure.zero(Radians);
+    public static MutableMeasure<Angle> UP_ANGLE = MutableMeasure.zero(Radians);
+=======
+    public static final MutableMeasure<Angle> DOWN_ANGLE = MutableMeasure.zero(Radians);
+    public static final MutableMeasure<Angle> UP_ANGLE = MutableMeasure.zero(Radians);
+>>>>>>> b38babce7c2115f5fefb513ee61988edc8bcbf20
+
+    public enum IntakePose {
```

Comment:
This enum should be in the intake class.

Reply:
but you said to put it in constants cyka

Reply:
381418fb60abfd384af6a39eaa907c7483a6ecf4

Reply:
IMO it's better to have it in the constants class, because it holds setpoints which are constants.

Reply:
was already changed
IMO

Reply:
8a21fe042ec15dac5f577d40cbb692c39c3aec95